app/scripts/chart-factory/google-chart-factory.js

```diff
@@ -7,70 +6,52 @@
  * # Extending Gobal Chart Factory for Google Chart Model
  *
  */
-
-angular.module('apacheZeppelinGsocApp').factory('GoogleChartFactory', function(ChartFactory) {
-
-
-  var ChartList = {'Bar':'BarChart','Line':'LineChart'};
+angular.module('apacheZeppelinGsocApp').factory('GoogleChartFactory', function(
+  ChartFactory) {
+  var ChartList = {
+    'Bar': 'BarChart',
+    'Line': 'LineChart'
+  };
   //googleChart model (sample chart data model)
   //TO-DO Sample Data will remove after model set.
   var GoogelChartChartModel = {
     type: 'BarChart',
-    cssStyle : 'height:400px; width:600px;',
-    data : {
-    'cols': [{
-    id: 'pizza',
-    label: 'Pizza',
-    type: 'string'
-}, {
-    id: 'populartiy',
-    label: 'Populartiy',
-    type: 'number'
-}
-
-    ],
-    'rows': [{
-c: [{
-    v: 'Pepperoni'
-}, {
-    v: 14
-}]
-    }, {
-c: [{
-    v: 'Mushroom'
-}, {
-    v: 6
-}]
-    }, {
-c: [{
-    v: 'Hawaiian'
-}, {
-    v: 5
-}]
-    }, {
-c: [{
-    v: 'Sausage'
-}, {
-    v: 10
-}]
-    }]
-},
-  options : {
-    'isStacked': 'true',
-    'fill': 20,
-    'height': 300,
-    'displayExactValues': true,
-    'vAxis': {
-    'gridlines': {
-    'count': 6
-}
+    cssStyle: 'height:400px; width:500px;',
+    data: {
```

Comment:
data and series will removed but options like height, width will be there.

On Wed, Jun 24, 2015 at 11:04 AM, CORNEAU Damien <notifications@github.com>
wrote:

> In app/scripts/chart-factory/google-chart-factory.js
> <https://github.com/Madhuka/apache-zeppelin-gsoc/pull/5#discussion_r33118408>
> :
>
> > -}, {
> > -    v: 10
> > -}]
> > -    }]
> > -},
> > -  options : {
> > -    'isStacked': 'true',
> > -    'fill': 20,
> > -    'height': 300,
> > -    'displayExactValues': true,
> > -    'vAxis': {
> > -    'gridlines': {
> > -    'count': 6
> > -}
> > +    cssStyle: 'height:400px; width:500px;',
> > +    data: {
>
> So is it going to disappear?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Madhuka/apache-zeppelin-gsoc/pull/5/files#r33118408>.
>



-- 
Cheers,
Madhuka Udantha
http://madhukaudantha.blogspot.com